src/items/itemDialog.component.tsx

```diff
@@ -843,14 +842,14 @@ function ItemDialog(props: ItemDialogProps) {
         );
       case 1:
         return (
-          <Grid item xs={12}>
+          <Grid xs={12}>
```

Comment:
```suggestion
          <Grid size={12}>
```
I believe one difference with Grid v2 is that it will inherit the columns and spacing (https://mui.com/material-ui/react-grid/#nested-grid). I think this might have something to do with why you needed columns={12} below, even though its the default value, so I think if you fix this size you may be able to remove the columns={12} below.

Alternatively it could be due to the width no longer growing by default (https://mui.com/material-ui/migration/upgrade-to-grid-v2/#container-width) in which case I found size={12} or width="100%" to be a better fix as opposed to changing how many columns it uses as most will assume its always the default 12.